### PR TITLE
Fix supervisor portal errors

### PR DESCRIPTION
**Pull request by bledsoef**
## Issue
Fixes issue #443 

There were some bugs and weird errors when using the supervisor portal like errors sorting, breaking presets, etc.

## Changes
- Removed evaluation functionality as that is no longer needed and throws errors.
- Stringified property attributes for Supervisor and Student sorting buttons to allow sorting to function properly.
- Added pre-student approval as a selected option to My Pending Forms.

## To Test
- Checkout `fix_supervisor_portal_errors`
- Verify that you can sort the results of form search by any of the columns.
- Make sure there are no traces of evaluations on the form search.
- Select My Pending Forms and verify that the "Pre-Student Approval" checkbox is automatically selected.

### CODE REVIEW

**Comment by bledsoef** [comment]
[Errors and Unwanted Form Search Behavior](https://trello.com/c/WcxsuYlP/818-errors-and-unwanted-form-search-behavior)

File: app/controllers/main_routes/main_routes.py [hunk 1]
```diff
@@ -205,7 +205,8 @@ def getDatatableData(request):
     
     else:
         filteredSearchResults = formSearchResults.order_by(colIndexColNameMap[sortColIndex]).limit(rowsPerPage).offset(rowNumber)
-
+    print(colIndexColNameMap[sortColIndex])
+    print("__________________________________________ FASFDDASF FIlterd search results", filteredSearchResults)
```
Comment (bledsoef) [inline app/controllers/main_routes/main_routes.py]: Delete this

Comment (WackyWeaver) [reply app/controllers/main_routes/main_routes.py]: We deleted them yesterday

**Comment by stevensonmichel** [comment]
It looks like when filtering for Approved Form Status, certain Terms and Academic Year such as Spring 2021 and AY 2021-2022 returns a warning message. 

Screenshot 2024-09-20 094547

That is something you should look into.
We are getting the same AJAX error when Denied Status Forms is filtered.

**Comment by WackyWeaver** [comment]
The AJAX error that Stevenson is getting is also on development. I tested this on the development branch, and it seems to throw the error on AY 2021-2022, AY 2020-2021, and Spring 2021.

**Comment by bledsoef** [comment]
@stevensonmichel the errors you are getting are on development as well and are due to the docker container's improper Tracy config. Can safely be ignored, excellent catch though.

**Comment by stevensonmichel** [comment]
Apparently, Cancel does not reset the state of the modal.
Screenshot 2024-10-04 102141

**Review by stevensonmichel — APPROVED** [review]

**Pull request merged**